apps/els_lsp/src/els_unused_includes_diagnostics.erl

```diff
@@ -87,26 +83,21 @@ expand_includes(Uri) ->
 expand_includes([], Graph, _Visited) ->
   Graph;
 expand_includes([Uri|Uris], Graph, Visited) ->
-  case els_dt_document:lookup(Uri) of
-    {ok, [Document]} ->
-      IncludedUris = els_diagnostics_utils:included_uris(Document),
-      NonVisitedIncludedUris = [U || U <- IncludedUris
-                                       , not sets:is_element(U, Visited)],
-      Dest = digraph:add_vertex(Graph, Uri),
-      NewGraph = lists:foldl(fun(U, G) ->
-                                 Src = digraph:add_vertex(G, U),
-                                 digraph:add_edge(G, Src, Dest),
-                                 G
-                             end
-                            , Graph
-                            , IncludedUris),
-      expand_includes( Uris ++ NonVisitedIncludedUris
-                     , NewGraph
-                     , sets:add_element(Uri, Visited));
-    {ok, []} ->
-      ?LOG_WARNING("Failed lookup while expanding includes [uri=~p]", [Uri]),
-      []
-  end.
+  {ok, Document} = els_utils:lookup_document(Uri),
```

Comment:
What would happen if the included path is incorrect or if it does not exist yet? Or, more likely, if the current project is not configured correctly, so the included file is not found? That could cause the provider to crash, right?